src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_params.py

```diff
@@ -143,5 +143,9 @@ def get_hostname_completion_list(prefix, action, parsed_args, **kwargs):  # pyli
 register_cli_argument('appservice web source-control', 'manual_integration', action='store_true', help='disable automatic sync between source control and web')
 register_cli_argument('appservice web source-control', 'repo_url', help='repository url to pull the latest source from, e.g. https://github.com/foo/foo-web')
 register_cli_argument('appservice web source-control', 'branch', help='the branch name of the repository')
+register_cli_argument('appservice web source-control', 'cd_provider', help='type of CI/CD provider', default='kudu', **enum_choice_list(['kudu', 'vsts']))
+register_cli_argument('appservice web source-control', 'cd_app_type', arg_group='VSTS CD Provider', help='web application framework you used to develop your app', default='AspNetWap', **enum_choice_list(['AspNetWap', 'AspNetCore', 'NodeJSWithGulp', 'NodeJSWithGrunt']))
+register_cli_argument('appservice web source-control', 'cd_account', arg_group='VSTS CD Provider', help='name of the Team Services account to create/use for continuous delivery', default='')
```

Comment:
I don't think you need `default=''`, as the custom method already has `None` as default. If `''` is the default value, please move to the method signature and add a comment for the meaning of `''`